testdata/source/ofx/test_vanguard_roth_ira_matching/training_examples.json

```diff
@@ -1,13 +1 @@
-[
-  [
-    {
-      "amount": "31.704 TYCDT",
-      "date": "2018-06-21",
-      "key_value_pairs": {
-        "ofx_type": "REINVEST"
-      },
-      "source_account": "Assets:Retirement:Vanguard:Roth-IRA:TYCDT"
-    },
-    "Income:Vanguard:Dividends:TYCDT"
```

Comment:
Why was this removed? I don't understand.

Reply:
Restored.

Reply:
I added `div_income` to `AUX_ACCOUNT_KEYS`, which causes these changes. This is because for treasury bills, the difference between the price of buying and seller should be counted as dividends.

I actually don't understand the meaning of `AUX_ACCOUNT_KEYS` and why it is needed to ensure the account isn't used for training though.

Reply:
>is because for treasury bills, the difference between the price of buying and seller should be counted as dividends.
That doesn't sound right to me. What's the logic?

The price can change even if no coupon payments have been made. Regular capital gain/loss.

Reply:
re: `AUX_ACCOUNT_KEYS` it looks like the logic is that since the user has manually provided auxiliary account keys, they must know what they're doing, and b-i shouldn't try to learn or infer.

I do agree agree `div_income` makes sense as part of `AUX_ACCOUNT_KEYS`. @jbms, is there anything we're missing here?

Reply:
I think the confusion is because I forgot to include a test case. It has been added and it should be clear.

Here is the base case: Treasury T-Bill (in contrast to Treasury bond.  T-Bill doesn't pay any coupon) sells at a discount: For example, you buy a bill at $98.00 (`buydebt`) and when it matures, the Treasury pays you $100 (`sellother` in ofx). The $2 difference goes to interests. Previously, I made a mistake in thinking it was dividends. I have fixed it in the PR and as a result, `training_examples.json` has been restored.

Now, you can sell a T-Bill before it matures. According to the reference, it seems that you will have additional capital gain/loss in addition to the interests if the selling price differs from the mature price. This PR does not support this case because I haven't never done this (no ofx for testing) and I am not sure if my understanding is 100% correct.

Reference: https://public.com/learn/how-are-treasury-bills-taxed#are-t-bills-taxed